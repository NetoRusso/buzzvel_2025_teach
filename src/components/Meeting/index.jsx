'use client'

import styles from "./styles.module.css";
import Image from "next/image";
import classNames from "classnames";
import Link from "next/link";
import React, { useState, useEffect, useRef } from 'react';

import photo_1 from "@/assets/picture_meeting_1.png";
import photo_2 from "@/assets/picture_meeting_2.png";
import photo_3 from "@/assets/picture_meeting_3.png";
import photo_4 from "@/assets/picture_meeting_4.png";
import photo_5 from "@/assets/picture_meeting_5.png";
import photo_6 from "@/assets/picture_meeting_6.png";
import photo_7 from "@/assets/picture_meeting_7.png";
import photo_8 from "@/assets/picture_meeting_8.png";
import photo_9 from "@/assets/picture_meeting_9.png";
import photo_10 from "@/assets/picture_meeting_10.png";

import plane from "@/assets/SVG/plane-departure.svg";
import arrow from "@/assets/SVG/arrow_right.svg";

const Meeting = () => {
  const [focusedPhoto, setFocusedPhoto] = useState(null);
  const intervalId = useRef(null);

  useEffect(() => {
    const changeFocus = () => {
      const randomIndex = Math.floor(Math.random() * 10) + 1;
      setFocusedPhoto(`content_photo_${randomIndex}`);
    };

    intervalId.current = setInterval(changeFocus, 1500);

    return () => clearInterval(intervalId.current);
  }, []);

  return (
    <section className={styles.container} aria-label="Meeting with teachers">
      <div className={styles.content}>
        <div className={styles.grid_pics}>
          <Image
            src={photo_1}
            alt="photo_1"
            style={{
              transform: focusedPhoto === 'content_photo_1' ? 'scale(1.05)' : 'scale(0.95)',
              opacity: focusedPhoto === 'content_photo_1' ? 1 : 0.8,
              transition: 'transform 0.3s ease-in-out, opacity 0.3s ease-in-out',
              zIndex: focusedPhoto === 'content_photo_1' ? 1 : 0,
            }}
            width={128}
            height={192}
            className={classNames(styles.content_photo, styles.content_photo_1)}
            loading="lazy"
          />
          <Image
            src={photo_2}
            alt="photo_2"
            style={{
              transform: focusedPhoto === 'content_photo_2' ? 'scale(1.05)' : 'scale(0.95)',
              opacity: focusedPhoto === 'content_photo_2' ? 1 : 0.8,
              transition: 'transform 0.3s ease-in-out, opacity 0.3s ease-in-out',
              zIndex: focusedPhoto === 'content_photo_2' ? 1 : 0,
            }}
            width={172}
            height={258}
            className={classNames(styles.content_photo, styles.content_photo_2)}
            loading="lazy"
          />
          <Image
            src={photo_3}
            alt="photo_3"
            style={{
              transform: focusedPhoto === 'content_photo_3' ? 'scale(1.05)' : 'scale(0.95)',
              opacity: focusedPhoto === 'content_photo_3' ? 1 : 0.8,
              transition: 'transform 0.3s ease-in-out, opacity 0.3s ease-in-out',
              zIndex: focusedPhoto === 'content_photo_3' ? 1 : 0,
            }}
            width={128}
            height={212}
            className={classNames(styles.content_photo, styles.content_photo_3)}
            loading="lazy"
          />
          <Image
            src={photo_4}
            alt="photo_4"
            style={{
              transform: focusedPhoto === 'content_photo_4' ? 'scale(1.05)' : 'scale(0.95)',
              opacity: focusedPhoto === 'content_photo_4' ? 1 : 0.8,
              transition: 'transform 0.3s ease-in-out, opacity 0.3s ease-in-out',
              zIndex: focusedPhoto === 'content_photo_4' ? 1 : 0,
            }}
            width={128}
            height={215}
            className={classNames(styles.content_photo, styles.content_photo_4)}
            loading="lazy"
          />
          <Image
            src={photo_5}
            alt="photo_5"
            style={{
              transform: focusedPhoto === 'content_photo_5' ? 'scale(1.05)' : 'scale(0.95)',
              opacity: focusedPhoto === 'content_photo_5' ? 1 : 0.8,
              transition: 'transform 0.3s ease-in-out, opacity 0.3s ease-in-out',
              zIndex: focusedPhoto === 'content_photo_5' ? 1 : 0,
            }}
            width={128}
            height={160}
            className={classNames(styles.content_photo, styles.content_photo_5)}
            loading="lazy"
          />
          <Image
            src={photo_6}
            alt="photo_6"
            style={{
              transform: focusedPhoto === 'content_photo_6' ? 'scale(1.05)' : 'scale(0.95)',
              opacity: focusedPhoto === 'content_photo_6' ? 1 : 0.8,
              transition: 'transform 0.3s ease-in-out, opacity 0.3s ease-in-out',
              zIndex: focusedPhoto === 'content_photo_6' ? 1 : 0,
            }}
            width={128}
            height={171}
            className={classNames(styles.content_photo, styles.content_photo_6)}
            loading="lazy"
          />
          <Image
            src={photo_7}
            alt="photo_7"
            style={{
              transform: focusedPhoto === 'content_photo_7' ? 'scale(1.05)' : 'scale(0.95)',
              opacity: focusedPhoto === 'content_photo_7' ? 1 : 0.8,
              transition: 'transform 0.3s ease-in-out, opacity 0.3s ease-in-out',
              zIndex: focusedPhoto === 'content_photo_7' ? 1 : 0,
            }}
            width={128}
            height={172}
            className={classNames(styles.content_photo, styles.content_photo_7)}
            loading="lazy"
          />
          <Image
            src={photo_8}
            alt="photo_8"
            style={{
              transform: focusedPhoto === 'content_photo_8' ? 'scale(1.05)' : 'scale(0.95)',
              opacity: focusedPhoto === 'content_photo_8' ? 1 : 0.8,
              transition: 'transform 0.3s ease-in-out, opacity 0.3s ease-in-out',
              zIndex: focusedPhoto === 'content_photo_8' ? 1 : 0,
            }}
            width={128}
            height={192}
            className={classNames(styles.content_photo, styles.content_photo_8)}
            loading="lazy"
          />
          <Image
            src={photo_9}
            alt="photo_9"
            style={{
              transform: focusedPhoto === 'content_photo_9' ? 'scale(1.05)' : 'scale(0.95)',
              opacity: focusedPhoto === 'content_photo_9' ? 1 : 0.8,
              transition: 'transform 0.3s ease-in-out, opacity 0.3s ease-in-out',
              zIndex: focusedPhoto === 'content_photo_9' ? 1 : 0,
            }}
            width={160}
            height={255}
            className={classNames(styles.content_photo, styles.content_photo_9)}
            loading="lazy"
          />
          <Image
            src={photo_10}
            alt="photo_10"
            style={{
              transform: focusedPhoto === 'content_photo_10' ? 'scale(1.05)' : 'scale(0.95)',
              opacity: focusedPhoto === 'content_photo_10' ? 1 : 0.8,
              transition: 'transform 0.3s ease-in-out, opacity 0.3s ease-in-out',
              zIndex: focusedPhoto === 'content_photo_10' ? 1 : 0,
            }}
            width={128}
            height={165}
            className={classNames(styles.content_photo, styles.content_photo_10)}
            loading="lazy"
          />
        </div>
        <div className={styles.content_text}>
          <h2 className={styles.content_text_title}>
            <Image src={plane} alt="plane" width={64} height={64} loading="lazy" className={styles.plane} />
            Meet international students & teachers
          </h2>
          <p className={styles.content_text_paragraph}>
            Morbi sit egestas dignissim pharetra, sed amet. Tempus justo senectus risus ac vel, velit, nunc. Eget commodo eget in aliquam facilisi facilisi nec magna hendrerit. Placerat ipsum sit tellus urna, faucibus aenean lorem faucibus integer.
          </p>
          <Link href="#" className={styles.content_link}>
            Explore teachers and students
            <Image src={arrow} alt="arrow" width={14} height={13.31} loading="lazy" className={styles.arrow} />
          </Link>
        </div>
      </div>
    </section>
  )
};

export default Meeting;