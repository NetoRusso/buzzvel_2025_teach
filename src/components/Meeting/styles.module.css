.container {
  width: 100%;
  height: auto;
  min-height: 100vh;
  display: flex;
  align-items: center;
  justify-content: center;
  padding: 16px 8px;
  margin-top: clamp(0rem, 32rem + -53.3333vw, -16rem);
}

.content {
  width: 100%;
  max-width: 1440px;
  height: auto;
  display: flex;
  flex-wrap: wrap-reverse;
  align-items: center;
  justify-content: space-around;
  gap: 40px;
}

.grid_pics {
  width: 90%;
  margin: 0 auto;
  max-width: 588px;
  min-width: 300px;
  display: grid;
  display: grid;
  grid-template-rows: repeat(6, 1fr);
  grid-template-columns: repeat(8, 1fr);
  justify-items: center;
  gap: 8px;
}

.content_photo {
  width: 100%;
  height: auto;
  object-fit: cover cover;
  object-position: center;
  background-repeat: no-repeat;
  transition: grid-area 0.3s ease-in-out;
}

.content_photo_1 {
  width: 80%;
  max-width: 128px;
  grid-area: 2 / 1 / 4 / 3;
  align-self: self-end;
}

.content_photo_2 {
  max-width: 172px;
  grid-area: 4 / 1 / 6 / 3;
  align-self: self-start;
}

.content_photo_3 {
  max-width: 128px;
  grid-area: 1 / 3 / 3 / 5;
}

.content_photo_4 {
  max-width: 128px;
  grid-area: 3 / 3 / 5 / 5;
}

.content_photo_5 {
  max-width: 128px;
  grid-area: 5 / 3 / 7 / 5;
}

.content_photo_6 {
  max-width: 128px;
  grid-area: 1 / 5 / 3 / 7;
}

.content_photo_7 {
  max-width: 128px;
  grid-area: 3 / 5 / 5 / 7;
}

.content_photo_8 {
  max-width: 128px;
  grid-area: 5 / 5 / 7 / 7;
}

.content_photo_9 {
  width: 80%;
  max-width: 160px;
  grid-area: 2 / 7 / 4 / 9;
}

.content_photo_10 {
  max-width: 128px;
  grid-area: 4 / 7 / 6 / 9;
}

.focused {
  z-index: 1;
  box-shadow: 0px 0px 8px 0px #00000012;
}

.focused.content_photo_1 {
  grid-area: 1 / 1 / 5 / 5;
}

.focused.content_photo_2 {
  grid-area: 3 / 1 / 7 / 5;
}

.focused.content_photo_3 {
  grid-area: 1 / 2 / 5 / 6;
}

.focused.content_photo_4 {
  grid-area: 2 / 2 / 6 / 6;
}

.focused.content_photo_5 {
  grid-area: 3 / 2 / 7 / 6;
}

.focused.content_photo_6 {
  grid-area: 1 / 4 / 5 / 8;
}

.focused.content_photo_7 {
  grid-area: 2 / 4 / 6 / 8;
}

.focused.content_photo_8 {
  grid-area: 3 / 4 / 7 / 8;
}

.focused.content_photo_9 {
  grid-area: 1 / 5 / 5 / 9;
}

.focused.content_photo_10 {
  grid-area: 3 / 5 / 7 / 9;
}

.content_text {
  width: 100%;
  max-width: 600px;
  min-width: 300px;
  display: flex;
  flex-flow: column nowrap;
  align-items: flex-start;
  justify-content: center;
  gap: 32px;
  padding: 16px;
}

.content_text_title {
  font-size: clamp(1.5rem, 0.9286rem + 2.8571vw, 3.5rem);
  line-height: 110%;
  font-weight: 800;
  color: var(--blue-Gray-900);
}

.content_text_title span {
  position: relative;
}

.plane {
  width: calc(24px + (64 - 24) * (100vw - 393px) / (1440 - 393));
  height: auto;
  position: absolute;
  top: -80%;
  left: 70%;
  z-index: -1;
}

.content_text_paragraph {
  font-size: clamp(1rem, 0.9062rem + 0.382vw, 1.25rem);
  line-height: 140%;
  font-weight: 400;
  color: var(--blue-Gray-900);
}

.content_link {
  font-size: clamp(1rem, 0.9062rem + 0.382vw, 1.25rem);
  line-height: 140%;
  font-weight: 400;
  color: var(--blue-600);
  display: flex;
  align-items: center;
  justify-content: flex-start;
  gap: 8px;
  transition: all 0.3s ease-in-out;
}

.content_link:hover {
  scale: 0.95;
  opacity: 0.9;
}