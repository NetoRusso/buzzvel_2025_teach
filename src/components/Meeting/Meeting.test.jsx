import React from 'react';
import { render, screen, act } from '@testing-library/react';
import '@testing-library/jest-dom';
import Meeting from './index';

import testImageImport from '@/assets/picture_meeting_1.png';


jest.mock('next/image', () => {
  const ImageComponent = (props) => {
    let { src: srcProp, alt, width, height, style, className, loading, ...rest } = props;

    let finalSrc = srcProp;
    if (typeof srcProp === 'object' && srcProp !== null && srcProp.src) {
      finalSrc = srcProp.src;
    }

    return <img src={finalSrc} alt={alt || ''} width={width} height={height} style={style} className={className} loading={loading || 'lazy'} {...rest} />;
  };
  ImageComponent.displayName = 'NextImageMock';
  return {
    __esModule: true,
    default: ImageComponent,
  };
});

jest.mock('next/link', () => {
  const LinkComponent = ({ children, href, ...props }) => <a href={href} {...props}>{children}</a>;
  LinkComponent.displayName = 'NextLinkMock';
  return {
    __esModule: true,
    default: LinkComponent,
  };
});


describe('Meeting Component', () => {
  let mockMath;
  const originalMath = global.Math;

  beforeEach(() => {
    jest.useFakeTimers();
    mockMath = Object.create(originalMath);
    global.Math = mockMath;
  });

  afterEach(() => {
    jest.clearAllTimers();
    jest.useRealTimers();
    global.Math = originalMath;
    jest.restoreAllMocks();
  });

  test('renders static content correctly', () => {
    render(<Meeting />);
    expect(screen.getByRole('heading', { name: /Meet international\s+plane\s+students & teachers/i })).toBeInTheDocument();
    expect(screen.getByText(/Morbi sit egestas dignissim pharetra/i)).toBeInTheDocument();
    const link = screen.getByRole('link', { name: /Explore teachers and students/i });
    expect(link).toBeInTheDocument();
    expect(link).toHaveAttribute('href', '#');

    const planeIcon = screen.getByAltText('plane');
    expect(planeIcon).toBeInTheDocument();
    expect(planeIcon).toHaveAttribute('src', '/img.jpg');

    const arrowIcon = screen.getByAltText('arrow');
    expect(arrowIcon).toBeInTheDocument();
    expect(arrowIcon).toHaveAttribute('src', '/img.jpg');

    expect(screen.getByRole('region', { name: 'Meeting with teachers' })).toBeInTheDocument();
  });

  test('renders all 10 photos initially with correct attributes and styles', () => {
    render(<Meeting />);
    for (let i = 1; i <= 10; i++) {
      const photo = screen.getByAltText(`photo_${i}`);
      expect(photo).toBeInTheDocument();
      expect(photo).toHaveAttribute('src', '/img.jpg');
      if (i === 1) {
        expect(photo).toHaveStyle('transform: scale(0.95)');
        expect(photo).toHaveStyle('opacity: 0.95');
      } else {
        expect(photo).toHaveStyle('transform: scale(0.95)');
        expect(photo).toHaveStyle('opacity: 0.8');
      }
      expect(photo).toHaveStyle('z-index: 0');
    }
  });

  test('focuses a photo after interval and applies correct styles', () => {
    mockMath.random = () => 0;
    render(<Meeting />);
    const photo1 = screen.getByAltText('photo_1');
    act(() => {
      jest.advanceTimersByTime(1500);
    });
    expect(photo1).toHaveStyle('transform: scale(1.1)');
    expect(photo1).toHaveStyle('opacity: 1');
    expect(photo1).toHaveStyle('z-index: 1');
  });

  test('changes focused photo over multiple intervals', () => {
    let randomCounter = 0;
    const randomValues = [0, 0.1, 0.2];
    mockMath.random = () => {
      const val = randomValues[randomCounter % randomValues.length];
      randomCounter++;
      return val;
    };
    render(<Meeting />);
    const photo1 = screen.getByAltText('photo_1');
    const photo2 = screen.getByAltText('photo_2');

    act(() => { jest.advanceTimersByTime(1500); });
    expect(photo1).toHaveStyle('transform: scale(1.1)');

    act(() => { jest.advanceTimersByTime(1500); });
    expect(photo1).toHaveStyle('transform: scale(0.95)');
    expect(photo2).toHaveStyle('transform: scale(1.05)');
  });

  test('clears interval on unmount', () => {
    const clearIntervalSpy = jest.spyOn(global, 'clearInterval');
    const { unmount } = render(<Meeting />);
    unmount();
    expect(clearIntervalSpy).toHaveBeenCalledTimes(1);
  });

  test('matches snapshot on initial render', () => {
    const { container } = render(<Meeting />);
    expect(container.firstChild).toMatchSnapshot();
  });

  test('matches snapshot when a photo is focused', () => {
    mockMath.random = () => 0.4;
    const { container } = render(<Meeting />);
    act(() => {
      jest.advanceTimersByTime(1500);
    });
    expect(container.firstChild).toMatchSnapshot();
  });
});