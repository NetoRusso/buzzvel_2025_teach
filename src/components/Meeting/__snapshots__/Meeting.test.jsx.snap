// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meeting Component matches snapshot on initial render 1`] = `
<section
  aria-label="Meeting with teachers"
  class="container"
>
  <div
    class="content"
  >
    <div
      class="grid_pics"
    >
      <img
        alt="photo_1"
        class="content_photo content_photo_1"
        height="192"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.95; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_2"
        class="content_photo content_photo_2"
        height="258"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="172"
      />
      <img
        alt="photo_3"
        class="content_photo content_photo_3"
        height="212"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_4"
        class="content_photo content_photo_4"
        height="215"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_5"
        class="content_photo content_photo_5"
        height="160"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_6"
        class="content_photo content_photo_6"
        height="171"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_7"
        class="content_photo content_photo_7"
        height="172"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_8"
        class="content_photo content_photo_8"
        height="192"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_9"
        class="content_photo content_photo_9"
        height="255"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="160"
      />
      <img
        alt="photo_10"
        class="content_photo content_photo_10"
        height="165"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
    </div>
    <div
      class="content_text"
    >
      <h2
        class="content_text_title"
      >
        Meet 
        <span>
          international 
          <img
            alt="plane"
            class="plane"
            height="64"
            loading="lazy"
            src="/img.jpg"
            width="64"
          />
        </span>
        students & teachers
      </h2>
      <p
        class="content_text_paragraph"
      >
        Morbi sit egestas dignissim pharetra, sed amet. Tempus justo senectus risus ac vel, velit, nunc. Eget commodo eget in aliquam facilisi facilisi nec magna hendrerit. Placerat ipsum sit tellus urna, faucibus aenean lorem faucibus integer.
      </p>
      <a
        class="content_link"
        href="#"
      >
        Explore teachers and students
        <img
          alt="arrow"
          class="arrow"
          height="13.31"
          loading="lazy"
          src="/img.jpg"
          width="14"
        />
      </a>
    </div>
  </div>
</section>
`;

exports[`Meeting Component matches snapshot when a photo is focused 1`] = `
<section
  aria-label="Meeting with teachers"
  class="container"
>
  <div
    class="content"
  >
    <div
      class="grid_pics"
    >
      <img
        alt="photo_1"
        class="content_photo content_photo_1"
        height="192"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.95; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_2"
        class="content_photo content_photo_2"
        height="258"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="172"
      />
      <img
        alt="photo_3"
        class="content_photo content_photo_3"
        height="212"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_4"
        class="content_photo content_photo_4"
        height="215"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_5"
        class="content_photo content_photo_5"
        height="160"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(1.05); opacity: 1; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 1;"
        width="128"
      />
      <img
        alt="photo_6"
        class="content_photo content_photo_6"
        height="171"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_7"
        class="content_photo content_photo_7"
        height="172"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_8"
        class="content_photo content_photo_8"
        height="192"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
      <img
        alt="photo_9"
        class="content_photo content_photo_9"
        height="255"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="160"
      />
      <img
        alt="photo_10"
        class="content_photo content_photo_10"
        height="165"
        loading="lazy"
        src="/img.jpg"
        style="transform: scale(0.95); opacity: 0.8; transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out; z-index: 0;"
        width="128"
      />
    </div>
    <div
      class="content_text"
    >
      <h2
        class="content_text_title"
      >
        Meet 
        <span>
          international 
          <img
            alt="plane"
            class="plane"
            height="64"
            loading="lazy"
            src="/img.jpg"
            width="64"
          />
        </span>
        students & teachers
      </h2>
      <p
        class="content_text_paragraph"
      >
        Morbi sit egestas dignissim pharetra, sed amet. Tempus justo senectus risus ac vel, velit, nunc. Eget commodo eget in aliquam facilisi facilisi nec magna hendrerit. Placerat ipsum sit tellus urna, faucibus aenean lorem faucibus integer.
      </p>
      <a
        class="content_link"
        href="#"
      >
        Explore teachers and students
        <img
          alt="arrow"
          class="arrow"
          height="13.31"
          loading="lazy"
          src="/img.jpg"
          width="14"
        />
      </a>
    </div>
  </div>
</section>
`;
